grails-app/controllers/org/pih/warehouse/api/InvoiceApiController.groovy

```diff
@@ -127,8 +128,23 @@ class InvoiceApiController {
         return invoice
     }
 
+    def getAllInvoiceItems() {
+        Location location = Location.get(session.warehouse.id)
+        params.partyFromId = location?.organization?.id
+        List<InvoiceList> invoices = invoiceService.getInvoices(params)
```

Comment:
One more point here - should we pass the whole params object as an argument as it is done here?
```
List<InvoiceList> invoices = invoiceService.getInvoices(params)
```